wallet/src/de/schildbach/wallet/WalletApplication.java

```diff
@@ -708,7 +710,14 @@ public void resetBlockchain() {
 
     private void resetBlockchainSyncProgress() {
         Executors.newSingleThreadExecutor().execute(() -> {
-            BlockchainState blockchainState = blockchainStateDao.loadSync();
+            BlockchainState blockchainState;
+
+            try {
+                 blockchainState = blockchainStateDao.loadSync();
+            } catch (SQLiteException ex) {
```

Comment:
https://console.firebase.google.com/u/1/project/dash-wallet-firebase/crashlytics/app/android:hashengineering.darkcoin.wallet/issues/b3e14b4cc0c76e13ee70518ec18239cc?time=last-seven-days&types=crash&sessionEventKey=62B66B8300BD00015FC8ED407818D6C2_1691436580845779912

Could be a disk error or something else. If we don't reset the percentage, the user won't see `Syncing...` and might get a CrowdNode error if he goes into the portal too quickly. Not critical.